src/mockConverter.ts

```diff
@@ -47,20 +47,26 @@ function getStringFakeValue({
 }) {
     casual.seed(hashedString(name + propertyName));
 
+    let value;
+
     if (!format) {
-        return `'${propertyName}-${name.toLowerCase()}'`;
+        // simple string
+        value = `'${propertyName}-${name.toLowerCase()}'`;
     } else if (format === StringFormats.Guid || propertyName === PropertyNames.Id) {
-        return `'${casual.uuid}'`;
-    } else {
-        switch (format) {
-            case StringFormats.DateTime: {
-                return `'2019-06-10T06:20:01.389Z'`;
-            }
-            case StringFormats.Date: {
-                return `'2019-06-10'`;
-            }
-        }
+        value = `'${casual.uuid}'`;
+    } else if (format === StringFormats.DateTime || format === StringFormats.TimeSpan) {
+        value = `'2019-06-10T06:20:01.389Z'`;
+    } else if (format === StringFormats.Date) {
+        value = `'2019-06-10'`;
+    } else if (format === StringFormats.Email) {
+        value = `'${casual.email}'`;
+    }
+
+    if (!value) {
+        value = 'TODO: FIX';
```

Comment:
maybe throw an error instead? like unsupported format ?

Reply:
@ardeois I thought about it in a first place, but in that case mocks will not be generated, and it will be hard to trace where issue happened (to throw a nice error we would need DTO name, propName, propType, etc). But with TODO it will be noticed in code-review

Reply:
ok then